Fuzz: Add progress bar during `creduce` processing

gmlwns2000 opened the pull request:
## Wait with a calm!

![image](https://user-images.githubusercontent.com/4879345/159932747-76b0bd04-093a-400e-adce-2d0e0d9b3888.png)

The `fuzz.py -i -r` took really long time, so I think the progress bar is defiantly needed. Progressbar helps users to understand how much the `creduce` is processed.

I tried to use `tqdm`, and I noticed `tqdm` is not installed on the server. So I implement simple progress bar logic in `fuzz.py`. I wanted add to ETA, but it was too time-consuming job to do during working on HW2 :(

## Modified git ignore

Also, I found the `creduce_bug_*` folders are not in `.gitignore`. So I added it on `.gitignore`.

minseongg commented on tests/.gitignore:
```diff
@@ -2,3 +2,4 @@
 /csmith-*
 /test*.c
 /reduce-criteria.sh
+/creduce_bug_*
```
Please add a newline

gmlwns2000 replied:
Fixed!

gmlwns2000 commented:
> Thanks for your work!
> 
> 1. Please rebase and update on the current repo.
> 2. We installed `tqdm` package on the server. Could you please modify implementation by using `tqdm`? (If it can be done in much simpler way)

I rebased the repository to upstream. And moved to `tqdm`. Now it shows ETA since it is the default feature of `tqdm`.

![image](https://user-images.githubusercontent.com/4879345/160083968-2206e428-2cf9-4434-8f9b-0ac579ddc15a.png)

minseongg commented:
Thanks for the work! I think this feature is very helpful.
